package.json

```diff
@@ -32,15 +32,23 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "babel-runtime": "^6.23.0",
+    "bem-classnames": "^1.0.7",
     "compass-importer": "^0.4.1",
     "dotenv": "^4.0.0",
     "esprima": "^3.1.3",
+    "fbemitter": "^2.1.1",
     "glob": "^7.1.1",
     "accessible-autocomplete": "git+https://github.com/mapseed/accessible-autocomplete.git",
     "node-env-file": "^0.1.7",
     "node-sass": "^4.0.0",
     "npm": "^5.2.0",
     "s3-website": "^3.2.2",
+    "prop-types": "^15.6.0",
+    "rc-color-picker": "^1.2.0",
+    "react": "^15.6.2",
```

Comment:
Let's upgrade these deps. React 16 is now available, so we should be using that.